### DISCUSSION TITLE
Implement the export feature when editing beatmaps in mobile

### DISCUSSION THREAD

**Discussion by PayToUse** [ppy/osu#25582] [Ideas]
What if peppy made a specific folder for exporting beatmaps after editing? Would it be possible to implement it in order to export them?

![Screenshot_20231126_161415_ru zdevs zarchiver](https://github.com/ppy/osu/assets/67496494/f5b46298-7cf7-4073-917e-2369e0b5dda3)

![Screenshot_20231126_160831_com gspace android](https://github.com/ppy/osu/assets/67496494/73a255b0-8927-46f7-ae02-fd65a0a8947f)

**Comment by nagi-desuuu** [comment]
Something something Android storage permissions thingy...
https://github.com/ppy/osu/issues/11911 (I could be wrong)

**Reply by PayToUse** [reply]
i forgot, that should be required
